visallo/visallo
# fix issue where document.body is not loaded fasted enough in ie11 which causes the table to not render correctly

Testing Instructions:
- Create a saved search with all of the data
- Open it in the saved search table
- Refresh the browser

Issue was in IE11, but make sure all supported browser still works

Points of Regression: N/A

NO CHANGELOG
